demisto_sdk/commands/pre_commit/pre_commit_context.py

```diff
@@ -52,10 +52,12 @@ def __post_init__(self):
         """
         shutil.rmtree(PRECOMMIT_FOLDER, ignore_errors=True)
         PRECOMMIT_FOLDER.mkdir(parents=True)
+        (PRECOMMIT_FOLDER / "coverage").mkdir()
+        (PRECOMMIT_FOLDER / "pytest-junit").mkdir()
         PRECOMMIT_CONFIG.mkdir()
         PRECOMMIT_DOCKER_CONFIGS.mkdir()
 
-        self.precommit_template = get_file_or_remote(PRECOMMIT_TEMPLATE_PATH)
+        self.precommit_template: dict = get_file_or_remote(PRECOMMIT_TEMPLATE_PATH)  # type: ignore[assignment]
```

Comment:
why not to use the `files` package here to retrieve remote files? either from git / github?

Reply:
We can, but let's do it in another PR